helpers/deploymentConfig.ts

```diff
@@ -9,39 +9,21 @@ interface BridgeConfig {
 const SEPOLIA_MULTISIG = "0x94fa6078b6b8a26f0b6edffbe6501b22a10470fb";
 
 export const preconfiguredAddresses = {
-  hardhat: {
-    VTreasury: "account:deployer",
-  },
   bsctestnet: {
-    VTreasury: "0x8b293600C50D6fbdc6Ed4251cc75ECe29880276f",
-    NormalTimelock: "0xce10739590001705F7FF231611ba4A48B2820327",
-    FastTrackTimelock: "0x3CFf21b7AF8390fE68799D58727d3b4C25a83cb6",
-    CriticalTimelock: "0x23B893a7C45a5Eb8c8C062b9F32d0D2e43eD286D",
-    AccessControlManager: "0x45f8a08F534f34A97187626E05d4b6648Eeaa9AA",
     XVS: "0xB9e0E753630434d7863528cc73CB7AC638a7c8ff",
```

Comment:
It should be available in the npm package @venusprotocol/venus-protocol, since v6.0.0

Reply:
[9d3a335](https://github.com/VenusProtocol/token-bridge/pull/2/commits/9d3a335283ddfbf9ba26a3f1fe81a008a114e21e)